src/EditorFeatures/CSharpTest/SignatureHelp/InvocationExpressionSignatureHelpProviderTests.cs

```diff
@@ -1107,7 +1107,7 @@ void M()
 public class Goo
 {
     [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Always)]
-    public static void Bar() 
+    public static void Bar()
```

Comment:
not a huge deal... but in general we reallllllly do not like existing tests changing their input values.